### DISCUSSION TITLE
An issue with peacock

### DISCUSSION THREAD

**Discussion by DanielChou0916** [idaholab/moose#25886] [Q&A Getting Started] [answered]
Hello
I have an issue with using Peacock on WSL in the Windows system.
After the installation process, I was trying to run the example file.
`mamba activate moose`
`cd ~/projects/moose/tutorials/darcy_thermo_mech/step01_diffusion/problems`
`~/projects/moose/python/peacock/peacock -i step1.i`
However, I received the message:

> qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.
Available platform plugins are: eglfs, minimal, minimalegl, offscreen, vnc, webgl, xcb.

When I try this:
`../darcy_thermo_mech-opt -i step1.i`
Everything is fine, it returns 'Solve Converged!'
The xeyes works well, and I also enabled the ''Disable access control'' when using VcXsrv, so I'm not sure what's going on :(
Does anyone have a similar experience?

**Comment by DanielChou0916** [comment, marked as answer]
Hello guys
I eventually found that after using this in the base env, then activate moose and run the codes, all is well :)
`sudo apt install libxcb-*`

**Reply by milljm** [reply]
I'll add this to our instructions! 🍻
